### PR TITLE
Add TransferCommand.php

### PR DESCRIPTION
**Pull request by ghost**
Transfer Command!

### CODE REVIEW

**Review by SOF3 — CHANGES_REQUESTED** [review]
---

File: src/pocketmine/command/defaults/TransferCommand.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	public function execute(CommandSender $sender, $currentAlias, array $args){
+		if($sender instanceof Player){
+
+			if(isset($args[1])){
+				$port = $args[1];
+			}else{
+				$port = 19132;
+			}
```
Comment (SOF3) [inline src/pocketmine/command/defaults/TransferCommand.php]: Ever heard of something called null coalescence?

File: src/pocketmine/command/defaults/TransferCommand.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        		$pk = new TransferPacket();
+        		$pk->address = $args[0];
+        		$pk->port = $port;
```
Comment (SOF3) [inline src/pocketmine/command/defaults/TransferCommand.php]: And what is the type of this value? Int or string?

File: src/pocketmine/command/defaults/TransferCommand.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		if($sender instanceof Player){
+
+			if(isset($args[1])){
+				$port = $args[1];
+			}else{
+				$port = 19132;
+			}
+
+        		$pk = new TransferPacket();
+        		$pk->address = $args[0];
```
Comment (SOF3) [inline src/pocketmine/command/defaults/TransferCommand.php]: You should never assume user input is valid...

File: src/pocketmine/command/defaults/TransferCommand.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        		$pk->port = $port;
+        		$sender->dataPacket($pk);
+
+        		Command::broadcastCommandMessage($sender, "Connected to " . $args[0] . ":" . $port);
+		}else{
+			$sender->sendMessage("please run this command in game.");
```
Comment (SOF3) [inline src/pocketmine/command/defaults/TransferCommand.php]: Translations?

**Review by SOF3 — CHANGES_REQUESTED** [review]
Not that this will be merged once it's done, but please at least fix your indentation.

**Comment by dktapps** [comment]
This should be done by plugins. The core will provide API only.

**Comment by AppleDevelops** [comment]
Oh my

**Comment by AryToNeX** [comment]
My opinion: you should provide /transferserver command 'cause it's available in single player.
Plugins will use the API to provide advanced functions such as ping before transfer, portals and so on.

**Comment by dktapps** [comment]
Interesting, didn't spot that. 👍

**Pull request closed without merging**